src/components/__tests__/MenuSelect.js

```diff
@@ -298,7 +295,7 @@ describe('custom default render', () => {
           :value="item.value"
           :selected="item.isRefined"
         >
-          {{item.label}}
+          {{item.label.toUpperCase()}}
```

Comment:
(As explained below) If value attribute and the textContent are the same, Vue 3 won't render the value attribute. It makes writing tests harder. To prevent it, I made this label uppercase. See the snapshot file to see its result.